sharing_portal/views.py

```diff
@@ -488,16 +488,39 @@ def artifact(request, artifact, version_slug=None):
     )
     launch_url = preserve_sharing_key(launch_url, request)
 
+    # We use this download URL instead of the one from trovi so we can
```

Comment:
This comment confused me. It looks like we _are_ using the download URL from Trovi, but we're just fetching it when the button is clicked rather than when the page loads (which is the right thing to do, imo).

Reply:
I meant that this uses a portal download link e.g. `experiment/share/uuid/download` rather than directly setting the `href` to trovi's content URL.

Reply:
Yes, I understood that after I scrolled down in the PR and saw the download function. Upon initially reading the comment and seeing that we're doing a `reverse` instead of calling directly to Trovi in this function, I thought that we would not be fetching the link from Trovi at all for some reason. I did not realize we were just proxying the Trovi call.